### DISCUSSION TITLE
[Feature Request]: 关于刷流——支持多规则

### DISCUSSION THREAD

**Discussion by lonelyman0108** [jxxghp/MoviePilot#3373] [讨论]
### 当前程序版本

1.4.1

### 运行环境

Docker

### 功能改进类型

插件

### 功能改进

希望支持多规则刷流
例如
A站只刷文件名中包含abc的free种
B站只刷文件名中包含bcd的2xfree种
目前的筛选以及管理策略都是全局的，希望可以支持多规则

**Comment by LoSiTeE** [comment]
同求

**Comment by 953786940** [comment]
同求
